README.md

```diff
@@ -220,3 +220,16 @@ bars = client.get_crypto_bars(request_params)
 bars.df
 
 ```
+
+### Options Trading (Beta) <a name="options-trading"></a>
+
+We're excited to support options trading! Use this section to read up on Alpaca's Beta trading capabilities.
+For more details, please refer to [our documentation page for options trading](https://docs.alpaca.markets/v1.1/docs/options-trading)
+
+```
```

Comment:
Instead of a fenced code block I would use block quotes as this is not code.
> We're excited to support options trading! Use this sec...

Reply:
Thank you! Addressed https://github.com/alpacahq/alpaca-py/pull/410/commits/9ff392ea6c11cc33ebb13a6a644639a4f841186c